pages/dashboard.js

```diff
@@ -290,10 +289,17 @@ export async function getServerSideProps({ req, res, locale, query }) {
   const { userid } = query
   setCookie('userid', userid, { req, res, maxAge: 60 * 6 * 24 })
 
-  // const aemContent = await queryGraphQL(getDashboardPage).then((result) => {
-  //   return result;
-  // });
-  // console.log(aemContent)
+  // Get mapped content from AEM
+  let aemContent
+  try {
+    aemContent = (await getDashboardContent()) || {}
```

Comment:
If you want this to be a bit cleaner (aka, not changing the code for a test); in the test you should be able tell the mock what it should return for this function. Something like this
```js
jest.mock('../../lib/aem/mapper', () => {
   getDashboardContent: () => {}
 })
 ```

Reply:
I can't seem to get to work in this way. I keep getting errors. Did you want to give it a try?

Reply:
Gave it a shot. I led you the wrong way it seems... sorry.
All you should need to do is add this to the top of the failing test, then you can remove the OR condition.
`getDashboardContent.mockReturnValueOnce({})`